thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomalydetection/alertFilterAutotune/BaseAlertFilterAutotune.java

```diff
@@ -0,0 +1,24 @@
+package com.linkedin.thirdeye.anomalydetection.alertFilterAutotune;
+
+import com.linkedin.thirdeye.datalayer.dto.AnomalyFunctionDTO;
+import com.linkedin.thirdeye.datalayer.dto.MergedAnomalyResultDTO;
+import com.linkedin.thirdeye.detector.email.filter.AlertFilter;
+import com.linkedin.thirdeye.detector.email.filter.AlphaBetaAlertFilter;
+import com.linkedin.thirdeye.detector.email.filter.BaseAlertFilter;
+import java.util.List;
+import java.util.Map;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+/**
+ * Created by ychung on 2/8/17.
+ */
+public abstract class BaseAlertFilterAutotune implements AlertFilterAutoTune{
```

Comment:
This BaseAlertFilterAutotune seems not used anywhere. In the refactory in returned AlertFilterAutoTune (the interface, which put this baseClass not necessary).